packages/ws-worker/src/api/claim.ts

```diff
@@ -1,14 +1,38 @@
+import crypto from 'node:crypto';
+import * as jose from 'jose';
 import { Logger, createMockLogger } from '@openfn/logger';
 import { ClaimPayload, ClaimReply } from '@openfn/lexicon/lightning';
+
 import { CLAIM } from '../events';
 
 import type { ServerApp } from '../server';
 
 const mockLogger = createMockLogger();
 
-// TODO: this needs standalone unit tests now that it's bene moved
-const claim = (app: ServerApp, logger: Logger = mockLogger, maxWorkers = 5) => {
+const verifyToken = async (token: string, publicKey: string) => {
```

Comment:
Hi @stuartc - this is where we verify the run token based on the lightning public key. The key comes from `WORKER_LIGHTNING_PUBLIC_KEY` as a base64 encoded string - you can trace this through from `src/start.ts` if you wish